java/gazelle/private/servermanager/BUILD.bazel

```diff
@@ -2,8 +2,24 @@ load("@rules_go//go:def.bzl", "go_library")
 
 go_library(
     name = "servermanager",
-    srcs = ["servermanager.go"],
-    data = ["//java/src/com/github/bazel_contrib/contrib_rules_jvm/javaparser/generators:Main"],
+    srcs = ["servermanager.go"] + select({
+        "//java/gazelle:embedded_server": [
+            "embedded.go",
+        ],
+        "//conditions:default": [
+            "insource-server.go",
+        ],
+    }),
+    data = select({
+        "//java/gazelle:embedded_server": [],
+        "//conditions:default": ["//java/src/com/github/bazel_contrib/contrib_rules_jvm/javaparser/generators:Main"],
+    }),
+    embedsrcs = select({
+        "//java/gazelle:embedded_server": [
+            ":javaparser_deploy_jar",
+        ],
+        "//conditions:default": [],
+    }),
```

Comment:
Does gazelle-go 1. like 2. generate these selects?

Reply:
No, it definitely doesn't like these selects. I think however that some selects are necessary if we want to avoid compiling the non-embedded version when we compile the embedded one, and this is the lowest-impact place I could find for them.